frontend/http/parser.go

```diff
@@ -20,6 +20,13 @@ type ParseOptions struct {
 	MaxScrapeInfoHashes uint32 `yaml:"max_scrape_infohashes"`
 }
 
+// Default parser config constants.
+const (
+	defaultMaxNumWant          uint32 = 100
+	defaultDefaultNumWant      uint32 = 50
+	defaultMaxScrapeInfoHashes uint32 = 50
```

Comment:
You can leave the type off the const. Constant numbers can actually be "untyped" in Go. It's slightly more flexible.